salt/modules/acme.py

```diff
@@ -421,3 +426,87 @@ def needs_renewal(name, window=None):
         window = int(window)
 
     return _renew_by(name, window) <= datetime.datetime.today()
+
+
+def revoke(name, reason=None):
+    """
```

Comment:
Needs a `.. version-added:: 3007` here

Reply:
I'm not sure I follow.

Reply:
@twangboy can you elaborate on what you're referring to?

Reply:
When you add a new module, or a new function to a module, or a new parameter to a function, you need to note the version that that new item was added. That is done using `.. version-added:: <the version>`.

For example: https://github.com/saltstack/salt/blob/master/salt/modules/file.py#L597

Reply:
ah, ok. i hadn't seen that.